### PR TITLE
USHIFT-1426: unpin openvswitch 4.13

### PR DESCRIPTION
**Pull request by dhellmann**
https://issues.redhat.com/browse/OCPBUGS-15948

Manual cherry-pick of #2035

### CODE REVIEW

File: packaging/rpm/microshift.spec [hunk 1]
```diff
@@ -294,7 +299,10 @@ systemctl enable --now --quiet openvswitch || true
 # Use Git command to generate the log and replace the VERSION string
 # LANG=C git log --date="format:%a %b %d %Y" --pretty="tformat:* %cd %an <%ae> VERSION%n- %s%n" packaging/rpm/microshift.spec
 %changelog
-* Mon May 01 2023 Doug Hellmann <dhellmann@redhat.com> 4.14.0
+* Fri Jul  21 2023 Zenghui Shi <zshi@redhat.com> 4.13.0
```
Comment (zshi-redhat) [inline packaging/rpm/microshift.spec:302]: nit: extra space between Jul and 21.

**Comment by zshi-redhat** [comment]
The change looks good to me.
@ggiguash would you like to take another look?
I'll lgtm at the end of day if no other comments.

**Comment by ggiguash** [comment]
@dhellmann , we also need to remove this code:
https://github.com/openshift/microshift/blob/release-4.13/scripts/image-builder/build.sh#L249-L252

**Comment by dhellmann** [comment]
> @dhellmann , we also need to remove this code: https://github.com/openshift/microshift/blob/release-4.13/scripts/image-builder/build.sh#L249-L252

Done.

**Pull request merged**